kubesae/utils.py

```diff
@@ -145,7 +145,42 @@ def count_backups(c, bucket_identifier='caktus-hosting-services-backups', profil
         print(f"{v:03d}: {k}\n")
 
 
+@invoke.task
+def scale_app(c, down=False, celery=False):
+    """ A utility to simplify scaling namespace app pods and optionally celery pods.
+
+    Developed primarily to assist with backup verifications. Following the instructions in
+    https://caktus.github.io/developer-documentation/reference/backups/
+
+    If the namespace is using a standard django-k8s celery deployment, you can specify --celery in the command to
+    scale the celery worker and beat appropriately.
+
+    Default behavior is non-destructive, if you accidentally call this task it will merely be a no-op on a
```

Comment:
Good point. I will remove magic numbers for the app containers, but should we still fix celery worker and beat, I've only ever seen those at 1 and 1. And if they should be different do they scale together or would `beat` normally stay at 1 and `workers` scale?

Reply:
Sounds good.
Good question; we could probably try to use a config var or something like that, but maybe it would be easier (for us and probably anyone who uses this project) if we explicitly print out "scaling celery-beat to 1 worker" or something like that. What do you think?

Reply:
That sounds right to me.